sofagym/AbstractEnv.py

```diff
@@ -372,8 +372,10 @@ def reset(self):
             obs, info
 
         """
-        self.close()
-        self.initialization()
```

Comment:
Is there a problem here if you delete initialization? In the past we restart the server to initialize a new scene (with other configuration, for example random position at the beginning of the episode)

Reply:
The part where new random positions for the goal and resetting parameters of the scene (past actions list, timer) is already included in the rest of this `reset()` function, I removed the initialization to omit unnecessary re-init of configs that do not change between episodes. I added only the part to clean the registry, reset the viewer, and choose a new random seed.

I also removed the close function because it closes the scene completely alongside the display window, which shouldn't happen after each episode. The close function also calls `close_scene()` to send the command to all clients to close, but since `close_scene()` is also called at the beginning of `start_scene()` there would be no need for duplication.
https://github.com/SofaDefrost/SofaGym/blob/d48ff771227aa78808e6ac8810e2f0aca41652a3/sofagym/rpc_server.py#L493-L511

Reply:
Ok perfect